embedding/csharp/Tizen.Flutter.Embedding/FlutterApplication.cs

```diff
@@ -51,6 +51,11 @@ public class FlutterApplication : CoreUIApplication, IPluginRegistry
         /// </summary>
         protected bool IsTopLevel { get; set; } = false;
 
+        /// <summary>
+        /// The renderer type of Flutter engine.
+        /// </summary>
+        protected FlutterDesktopRendererType RendererType { get; set; } = FlutterDesktopRendererType.kEGL;
```

Comment:
What if the selected type is not supported by the current profile? (i.e. wearable)

How do we automatically select the `kEvasGL` type for wearable apps?

Reply:
I added profile check macro for this.